files/views.py

```
@@ -680,6 +681,171 @@ def upload_media(request):
+        for data in licenses_dict.values()
+    ]
+
+
+def _edit_media_page_config(request, form, media, licenses_dict, video_extensions):
+    form_fields = set(form.fields.keys())
+    media_file = _file_info(media.media_file)
+    uploaded_poster = _file_info(media.uploaded_poster)
+
+    try:
+        sprites_url = media.sprites.url if media.sprites else ""
+    except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)